modules/storages/spec/services/storages/one_drive_managed_folder_sync_service_spec.rb

```diff
@@ -256,6 +255,21 @@
                         message: nil,
                         data: { status: 400, body: /drive id/ })
         end
+
+        it "does not break in case of timeout", vcr: "one_drive/sync_service_root_read_failure" do
```

Comment:
❓ shouldn't you remove the `vcr` attribute here? No cassette is recorded, is it?

Reply:
There is a request to get token prior to the problematic one. So, I reused existing vcr cassette because of that.

Reply:
That makes sense. 👍🏽